### ISSUE TITLE
New car graphics

### ISSUE THREAD

**Issue by juzzlin** [juzzlin/DustRacing2D#35]
Current car graphics are aged. New, more modern ones needed.

**Comment by 0bl0ng0** [comment]
I have some experience with medium-small 2D pixelated sprites, so if is this what you are looking for, I hope I can help (same for the grass, tree, asphalt graphics and menu graphics)

**Comment by juzzlin** [comment]
Yes, please :) If you think you can improve the current graphics then go ahead. I'd like to have the source SVG's, but if drawing pixel graphics directly is easier I guess that's also ok.

**Comment by 0bl0ng0** [comment]
It's not necessary; I just have to know if the size and colour depth of the actual cars images are OK: 
175x93 pixels and 24 BPP.

**Comment by juzzlin** [comment]
Yeah, that's ok.

**Comment by aaferrari** [comment]
@juzzlin What do you think of using sprites like those that can be found in the following sites?:
https://opengameart.org/art-search?keys=racing
https://kenney.nl/assets?s=racing
https://www.reinerstilesets.de/graphics/2d-grafiken/2d-vehicles/
https://www.sharecg.com/b/1/Art/?attribute[2015][2191]=1
http://graphicsforgames.com/product-tag/racing/
Many of them have a Creative Commons or GPL license so it should not be inconvenient to include them in the game and in the long run you could also find assets for #36.

**Comment by juzzlin** [comment]
Thanks for the links! Maybe some of those sprites could be used in the game.

**Comment by tempral** [comment]
The actual car's color is too white
